adoc/chapters/programming_interface.adoc

```diff
@@ -1888,6 +1888,15 @@ _Remarks:_ Template parameter to [api]#device::get_info#.
 
 _Returns:_ A unique vendor device identifier.
 
+If the vendor has a PCI vendor ID, the low 16 bits must contain that PCI vendor
+ID, and the remaining bits must be set to zero.
+Otherwise, the value returned must be a valid Khronos vendor ID.
+Khronos vendor IDs are allocated starting at 0x10000, to distinguish them from
+the PCI vendor ID namespace.
+
+{note}Khronos vendor IDs are synchronized across APIs by utilizing Vulkan’s
+vk.xml as the central Khronos vendor ID registry.{endnote}
```

Comment:
If a vendor decides to support SYCL on a device that is not supported by Vulkan, would that vendor still update the "vk.xml" file?  I assume the answer would have to be "yes"?

Reply:
Not necessarily.  Vendors have two options:
1. Get a PCI vendor ID (see https://pcisig.com/membership/member-companies); or
2. Get a Khronos vendor ID, via the vk.xml file.

Option 1. is much more likely -- there are only currently [eight vendors with Khronos vendor IDs](https://raw.githubusercontent.com/KhronosGroup/Vulkan-Docs/refs/heads/main/xml/vk.xml).  I don't think supporting Vulkan is a requirement, rather that Khronos has decided to use Vulkan's XML file as the source of truth.  It doesn't even look like you have to have a "real" physical device: PoCL is an OpenCL implementation, and they have a Khronos vendor ID.